pkgs/by-name/we/webrtc-audio-processing/package.nix

```diff
@@ -62,7 +62,7 @@ stdenv.mkDerivation (finalAttrs: {
       with lib.platforms;
       lib.intersectLists
         # https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/webrtc/rtc_base/system/arch.h
-        (arm ++ aarch64 ++ mips ++ power ++ riscv ++ x86 ++ loongarch64)
+        (arm ++ aarch64 ++ mips ++ power ++ riscv ++ x86_64 ++ loongarch64)
```

Comment:
Comment needs to be updated.

Reply:
Upstream is fixing it, actually, I've moved it to badPlatforms instead for now.